maintain.js

```diff
@@ -1,25 +1,345 @@
-/* Copyright © 2021 Ríona Stokes and other contributors, MIT License. */
-'use strict'
+module.exports = () => {
```

Comment:
explicitly name the function: `Maintain`

Reply:
Resolved with commit d89dfeace145e064fabc2302d04f27258bc65513